project.clj

```diff
@@ -37,6 +37,7 @@
                  [ring-webjars "0.2.0"]
                  [ring/ring-core "1.8.0"]
                  [ring/ring-defaults "0.3.2"]
+                 [ring/ring-json "0.5.0"]
```

Comment:
also, we need a new version number for this one! (0.3.3-SNAPSHOT)

Reply:
Done, updated readme.md and history.md